docs/13.DEBUGGER-TRANSPORT.md

```diff
@@ -0,0 +1,238 @@
+# JerryScript debugger transport interface
+
+The transport interface support allows dynamic selection of transportation
+layers which can encode/decode or send/receive messages transmitted between
+the debugger client and server.
+
+**Warning** This API is not part of the standard JerryScript API and may change
```

Comment:
IMHO this comment is unnecessary, because this is not part of the public API. Private functions could be changed anytime (and anywhere) in the whole project.